### DISCUSSION TITLE
Compatibility with MagVenture model x100

### DISCUSSION THREAD

**Discussion by Ali750n** [umair-hassan/best-toolbox#39] [General]
Hi, I am new to BEST and I am intending to work with a TMS device with the help of this toolbox. Our TMS device is a **MagPro X100** from **MagVenture** company. Is this toolbox compatible with our TMS device? (We want to use the EMG part of that specific device)
If **YES**; is there any specific caution we should consider?
If **NO**; is there an alternative way to use it?

**Comment by umair-hassan** [comment]
Dear Ali,

Thanks for starting this discussion thread. Yes, MagVenture X100 is compatible as output device in the BEST toolbox. 

You discussed about EMG system, were you talking about the EMG from
MEP monitor of MagVenture? Or something else? This will help me in giving you some concrete advice.

Moreover, what kind of TMS protocols are you planning to do? Single pulse? Paired pulse? rTMS? etc

Thank you!
Umair Hassan

**Comment by Ali750n** [comment]
Thanks for the quick reply!
Yes, I was talking about the EMG from MEP monitor of MagVenture. And we are going to use the Single-Pulse protocol.
I would be grateful to hear your advice!

Thank you,
Ali Shahbazi

**Reply by umair-hassan** [reply]
Dear Ali,

Unfortuately MEP monitor of magventure does not have any interface to stream the time series EMG data out of their console application. I would be happy to work on this if there is any low level feature available from Magventure.

If you can coordinate and provide me with amy info that can stream this data, that would be great! 

Otherwise I am afraid it wont be possible with MEP monitor. 

Do you want to ask them about this data streaming possibility though? 

Thank you!
Umair Hassan
